client/blocks/login/login-form.jsx

```diff
@@ -762,6 +766,11 @@ export class LoginForm extends Component {
 		const isOauthLogin = !! oauth2Client;
 		const isPasswordHidden = this.isUsernameOrEmailView();
 		const isCoreProfilerLostPasswordFlow = isWooCoreProfilerFlow && currentQuery.lostpassword_flow;
+		const isFromAutomatticForAgenciesReferralClient =
+			isA4AOAuth2Client( oauth2Client ) &&
+			currentQuery &&
+			currentQuery.redirect_to &&
+			currentQuery.redirect_to.includes( '/client/' );
```

Comment:
Wondering what will happen if, in the future, we have a route on the agency side that could have that slug `*/client/*`. 

Also, if we introduce a Dashboard page for the Client Portal in the base route `/client`, without the final slash, that conditional won't work.